ci-operator/step-registry/ipi/conf/aws/marketplace/ipi-conf-aws-marketplace-commands.sh

```
@@ -81,12 +85,15 @@ function search_marketplace_image() {
   do
     local v_xy="${major}${v}"
     echo "Checking ${v_xy} ..."
-    jq --arg r "${v_xy}" '.Images[] | select(.Description | test($r))' "$aws_marketplace_images" | jq -s | jq -r '. | sort_by(.CreationDate) | last' > $selected_image
-    image_id=$(jq -r '.ImageId' $selected_image)
+    if ! jq --arg r "${v_xy}" '.Images[] | select(.Description | test($r))' "$aws_marketplace_images" | jq -s | jq -r '. | sort_by(.CreationDate) | last' > $selected_image; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)